<?xml version="1.0"?>
<ui version="4.0">
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Form</class>
 <widget class="QGroupBox" name="groupConfig">
       <property name="title">
        <string>Configuration</string>
       </property>
       <layout class="QGridLayout" name="gridConfig">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLeft">
          <item row="0" column="0">
           <widget class="QComboBox" name="comboObjective">
            <property name="toolTip">
             <string>The objective being used. Affects config file selection</string>
            </property>
            <item>
             <property name="text">
              <string>Mitutoyo 10X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mitutoyo 5X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mitutoyo 20X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mitutoyo 2X</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelObjective">
            <property name="toolTip">
             <string>The objective being used. Affects config file selection</string>
            </property>
            <property name="text">
             <string>Objective</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboRate">
            <property name="toolTip">
             <string>The A-line rate of the camera</string>
            </property>
            <item>
             <property name="text">
              <string>76 kHz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>146 kHz</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelRate">
            <property name="toolTip">
             <string>The A-line rate of the camera</string>
            </property>
            <property name="text">
             <string>A-line rate</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QComboBox" name="comboApodization">
            <property name="toolTip">
             <string>The type of window used to apodize the A-line spectra prior to FFT</string>
            </property>
            <item>
             <property name="text">
              <string>Hanning</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blackman</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hamming</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelApodization">
            <property name="toolTip">
             <string>The type of window used to apodize the A-line spectra prior to FFT</string>
            </property>
            <property name="text">
             <string>Apodization window</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridRight">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridProc">
            <item row="1" column="0">
             <widget class="QCheckBox" name="checkInterpolation">
              <property name="toolTip">
               <string>If true, linear-in-wavenumber values are interpolated (1st order) from linear-in-wavelength spectra</string>
              </property>
              <property name="text">
               <string>Lambda-k interpolation</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkFFT">
              <property name="toolTip">
               <string>If true, data is complex-valued signal</string>
              </property>
              <property name="text">
               <string>Fourier transform</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="checkBox">
              <property name="toolTip">
               <string>The precision of the FFT and interpolation. Affects processing speed!</string>
              </property>
              <property name="text">
               <string>32 bit (default 64)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
 <pixmapfunction></pixmapfunction>
 <connections/>
</ui>
